automated/linux/ssuite/ssuite-bench.yaml

```diff
@@ -0,0 +1,77 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: S-suite
+    description: "The S-suite is a suite of I/O-performance benchmarks.
+                  This test uses the S-suite to run, for every I/O scheduler
+                  available for the target device, two main benchmarks:
+                  1) A throughput benchmark, which measures throughput
+                     with random and sequential sync I/O. In this respect,
+                     sync I/O is the kind of I/O for which it is most
+                     difficult to reach a higher throughput.
+                  2) A responsiveness benchmark, which measures the start-up
+                     time of various popular application, in the presence
+                     of sequential I/O in the background (sequential
+                     is the kind of background I/O that makes it more
+                     difficult to guarantee a high responsiveness"
+    maintainer:
+        - paolo.valente@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+        - openembedded
+    scope:
+        - I/O performance
+    devices:
+        - x86
```

Comment:
The test definition seems generic, I am wondering does the tests from S suite work on ARM :) 

I had a try on moonshot arm64, but hit the following error. It works fine with x86 box.
```
+ ./run_main_benchmarks.sh 'throughput replayed-startup'
+ tee -a /root/github/Linaro/test-definitions/automated/linux/ssuite/output/result.txt
No user config found in /root, copying default config
Block devices for partition sda2 unrecognized.

root@m1c15:~/github/Linaro/test-definitions/automated/linux/ssuite# df
Filesystem     1K-blocks     Used Available Use% Mounted on
udev            32935132        0  32935132   0% /dev
tmpfs            6596020    41384   6554636   1% /run
/dev/sda2      396522088 41430936 334879236  12% /
tmpfs           32980096        0  32980096   0% /dev/shm
tmpfs               5120        0      5120   0% /run/lock
tmpfs           32980096        0  32980096   0% /sys/fs/cgroup
/dev/sda1         524008      160    523848   1% /boot/efi
tmpfs            6596016        0   6596016   0% /run/user/0

```